### DISCUSSION TITLE
Is it possible to not include a sub dir in URL?

### DISCUSSION THREAD

**Discussion by frederikhors** [sveltejs/kit#2954] [Q&A] [answered]
I have these routes:

```
1. src/routes
  1. /players (localhost:3000/players)
  1. /teams (localhost:3000/teams)
```

Is there a way to have this instead?

```
1. src/routes
  1. /game
    1. /players (localhost:3000/players)
    1. /teams (localhost:3000/teams) 
```

Note the link is the same as before despite the new dir "game"

Is this possible?

**Comment by Theo-Steiner** [comment]
No, there is no straight forward way to do this. 
I assume you want the directory for the purpose of organizing your code but don't want to give up on the clean URL?
One way I think you could achieve this is if you have both routes as components that live under ``src/lib/game/`` → ``src/lib/game/players.svelte`` && ``src/lib/game/teams.svelte`` and then in ``src/routes`` you have a parameterized route ``src/routes/[game].svelte``, in which you either load ``teams.svelte`` or ``players.svelte`` based on the [slug] you receive.

**Reply by frederikhors** [reply]
> I assume you want the directory for the purpose of organizing your code but don't want to give up on the clean URL?

Yep.

I think this solution is an hacky one.

I'll wait for the future of this brilliant project!

**Comment by ignatiusmb** [comment, marked as answer]
No. You can follow [this feature request](https://github.com/sveltejs/kit/issues/3832) until then.
